libfoundation/src/system-file-posix.cpp

```diff
@@ -364,8 +364,6 @@ __MCSFileSetContents (MCStringRef p_native_path,
 
 	if (0 != rename (*t_temp_path_sys, *t_path_sys))
 	{
-		int t_save_errno = errno;
-
```

Comment:
@montegoulding I usually read the diff before pushing to make sure I haven't touched lines that I didn't mean to.  It seems Xcode is very aggressive about screwing with whitespace in nearby lines.